src/contract/table-cursor.ts

```diff
@@ -75,7 +75,7 @@ export class TableCursor<TableRow> {
      *
      * @returns The new rows.
      */
-    async next(): Promise<TableRow[]> {
+    async next(rowsCount?: number): Promise<TableRow[]> {
```

Comment:
Let me know what you think of this. I feel like you should have to define the limit here if you only want to fetch a specific number of rows per call.

Reply:
I like this and think it makes sense!